### PR TITLE
Apply ESLint and Prettier

### LINKED ISSUE #11: Apply ESLint

### PR DESCRIPTION
**Pull request by anthonyminyungi**
- It closes #11

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,6 +47,16 @@
...
+    "@typescript-eslint/eslint-plugin": "^4.6.1",
+    "@typescript-eslint/parser": "^4.6.1",
+    "eslint-config-airbnb": "^18.2.1",
+    "eslint-config-airbnb-typescript": "^12.0.0",
+    "eslint-config-prettier": "^6.15.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
+    "eslint-plugin-prettier": "^3.1.4",
+    "eslint-plugin-react": "^7.21.5",
+    "prettier": "^2.1.2"
```
Comment (anthonyminyungi) [inline package.json:60]: `devDependencies` that needed for formatting.

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "extends": [
+    "airbnb",
+    "airbnb/hooks",
+    "prettier/react",
+    "plugin:@typescript-eslint/recommended",
+    "prettier/@typescript-eslint",
+    "airbnb-typescript",
+    "plugin:prettier/recommended",
+    "prettier"
```
Comment (anthonyminyungi) [inline .eslintrc:10]: ESLint options.

File: .prettierrc [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  "singleQuote": true,
+  "semi": true,
+  "useTabs": false,
+  "tabWidth": 2,
+  "trailingComma": "all",
+  "printWidth": 80,
+  "arrowParens": "always",
+  "orderedImports": true,
+  "bracketSpacing": true,
+  "jsxBracketSameLine": false
```
Comment (anthonyminyungi) [inline .prettierrc:11]: Prettier options.

**Review by anthonyminyungi — COMMENTED** [review]
Other changes that have no comments are just caused by formatting.

**Review by HyeockJinKim — APPROVED** [review]
LGTM

**Pull request merged**